src/app/(private)/(docs)/docs/drift/databaser/create-postgresql-database/page.md

```diff
@@ -0,0 +1,76 @@
+---
+tittel: Hvordan lage en PostgreSQL database for et prosjekt
+---
+
+_skrevet 07.11.2024_
+
+# Hvordan lage en PostgreSQL database for et prosjekt
+
+## Div. info
+
+- Å skrote rundt med databaser er svært risikabelt så her skal man helst være to stykker så man forsikrer seg at ingen uskyldige databaser går tapt.
+- Alle databasene som driftes av Drift (pun intended) ligger på "_database-server_"-instansen på OpenStack.
+- Av sikkerhetsgrunner kan man bare koble til databaser fra NTNU sitt nettverk for å slippe at angripere fra Russland og lignende steder kan koble seg på. Trenger du tilgang til databasen utenfor NTNU sitt nett så må du bruke NTNU sin VPN.
+
+## Det du trenger
+
+Du trenger bare SSH-tilgang til _database-server_-instansen. Har du ikke det så kan du spørre folk i Drift til du finner noen som har det. Du skal også helst ha en kompis å kjøre kommandoene med fordi database-greier er risikabelt arbeid.
+
+## Lag databasen automagisk (anbefalt)
+
+Drift har laget et shell-script som lager databasen for deg. For å bruke dette scriptet SSH-er du deg inn på database serveren og kjører
+
+```bash
+create-postgresql-database <database-navn>
+```
+
+hvor `<database-navn>` er navnet på databasen du skal lage. Databasen vil da bli laget automagisk og du vil da få ut credentials som du må ta vare på og ikke miste(!!!).
+
+_HUSK: du kan bare koble til databasen fra NTNU sitt interne nettverk. Bruk NTNU sin VPN for å få tilgang fra utsiden av NTNU sitt nettverk._
```

Comment:
Du kan erstatte dette med:
```
{% callout title="HUSK" type="warning" %}
Du kan bare koble til databasen fra NTNU sitt interne nettverk. Bruk NTNU sin VPN for å få tilgang fra utsiden av NTNU sitt nettverk.
{% /callout %}
```

Da blir det sånn nice card, og det er lettere for folk å ikke overse det ;))

Reply:
fixed